emanate/__init__.py

```diff
@@ -10,35 +10,18 @@
 
 In `clean` mode, emanate instead removes such symbolic links.
 
-Examples:
-    Emanate defaults to using the current directory as source,
-    and the current user's home directory as destination::
-
-        ~/.dotfiles $ emanate
-
-    would create symbolic links in ~ for files in ~/.dotfiles
-
-    Emanate also defaults to looking for a configuration file in the source
-    directory, allowing usages such as::
-
-        $ cat software/foo/emanate.json
-        { 'destination': '/usr/local' }
-
-        $ emanate --source software/foo
-        ${PWD}/software/foo/bin/foo -> /usr/local/bin/foo
-        ${PWD}/software/foo/lib/library.so -> /usr/local/lib/library.so
-
-    See `emanate --help` for all command-line options.
-
 """
 
-from argparse import ArgumentParser, SUPPRESS
+from collections import namedtuple
 from fnmatch import fnmatch
 from pathlib import Path
 import sys
 from . import config
 
 
+FilePair = namedtuple('FilePair', ['src', 'dest'])
```

Comment:
My preference would be to use `typing.NamedTuple` or `@dataclasses.dataclass`, depending on what compatibility you wanted.

Reply:
I have no idea what are the various options, I just used the first one that came up when searching for “Python 3 named tuple”  :innocent: